### PR TITLE
Minor fixes to test cases, readme and formatter

### LINKED ISSUE #138: Problems in ModelRepositoryTest.java
Problems occur when trying to run the ModelRepositoryTest class.
It seems that `Guice.createInjector(..) ` throws an ArrayOutOfBoundsException.
Could not solve the problem with first search engine results (e.g. raise guice version to 4.1.0)
For now I ignored the whole test class.

### PR DESCRIPTION
**Pull request by martin-fleck-at**
- Fixes #138: Fix ModelRepositoryTest and remove ignore
- Avoid empty line conflict between Save/Clean action and formatter
- Fix and restructure README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,31 @@
...
-- `org.eclipse.m2e.core.prefs`
-
-
-*Checkstyle*<br>
-To configure Checkstyle for the new project right click on the project, choose `Checkstyle > Configure project(s) from blueprint...` and select `com.eclipsesource.modelserver.common` as blueprint project.
-Run `Checkstyle > Check Code with Checkstyle` to make sure Checkstyle is activated correctly.
+The code coverage report is generated with [JaCoCo](https://www.eclemma.org/jacoco/) and is integrated in the Maven build. In the package `com.eclispesource.modelserver.codecoverage` all code coverages are aggregated into one report.
 
-Please make sure to include the `.settings` folder as well as the `.checkstyle` settings file to the repository in your initial commit.
+When executing the Maven build executed locally, the detailed results are computed and can be investigated in more detail.
```
Comment (ndoschek) [inline README.md]: Executing once should be enough ;)

File: README.md [hunk 2]
```diff
@@ -150,25 +128,25 @@ public interface ModelServerClientApiV1<A> {
...
 // perform simple GET
-client.get("Coffee.ecore")
-      .thenAccept(response -> System.out.println("GET: " + response.body()));
+client.get("SuperBrewer3000.json")
+      .thenAccept(response -> System.out.println(response.body()));
 
-// perform same GET, but expect an EObject
-client.get("Coffee.ecore", "xmi")
-      .thenAccept(response -> System.out.println("GET: " + response.toString()));
+// perform same GET, but expect XMI format
```
Comment (ndoschek) [inline README.md]: Should be EObject not XMI format

File: README.md [hunk 3]
```diff
@@ -324,15 +301,41 @@ client.subscribe(subscriptionId, new JsonToEObjectSubscriptionListener() {
             System.err.println("Cannot decode incremental update: " + e.getMessage());
         }
     }
-}, "json");
+});
```
Comment (ndoschek) [inline README.md]: Please re-add the `EditingDomain` line and `"json"` format parameter again.

**Review by ndoschek — CHANGES_REQUESTED** [review]
Thanks a lot for the fixes!! Looks great to me! Only spotted 3 minor issues in the Readme file.

**Comment by martin-fleck-at** [comment]
@ndoschek Good catch, I fixed the README again ;-)

**Review by ndoschek — APPROVED** [review]

**Pull request merged**